### PR TITLE
T3950 create a docker image

### PR DESCRIPTION
**Pull request by parabolic**
- make it as universal as possible
- specify everything as env variables.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM golang:latest
```
Comment (parabolic) [inline Dockerfile]: No particular reason. I think in this case it doesn't hurt to use the latest.

File: License.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Savedo GmbH
```
Comment (erhangundogan) [inline License.md]: email missing

File: Dockerfile (hunk 1 above)
Comment (erhangundogan) [inline Dockerfile]: there should be a particular reason to use `golang` image IMHO. But i don't know the necessity so i cannot tell anything..

File: License.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (erhangundogan) [inline License.md]: this should not be markdown file
and file name should be all capital letters eg `LICENSE`

File: Dockerfile (hunk 1 above)
Comment (parabolic) [inline Dockerfile]: what do you mean, this proxy runs in go, I cannot use any other image ?

File: License.md (hunk 2 above)
Comment (parabolic) [inline License.md]: not necessary 
https://opensource.org/licenses/MIT

**Pull request merged**